hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -8787,7 +8787,10 @@ public void logAuditEvent(boolean succeeded, String userName,
         sb.setLength(0);
         sb.append("allowed=").append(succeeded).append("\t")
             .append("ugi=").append(userName).append("\t")
-            .append("ip=").append(addr).append("\t")
+            // InetAddress#tostring can also return hostname in addition to IP address.
+            // To not include hostname regardless of the hostname resolution, we should
+            // only include IP address here. See HADOOP-18628.
```

Comment:
I think this change is independent of HADOOP-18628. So we don't need to refer to it.
```suggestion
            // only include IP address here.
```

Reply:
Done